tool/tctl/common/collection.go

```diff
@@ -1164,6 +1165,29 @@ func (c *botCollection) writeText(w io.Writer, verbose bool) error {
 	return trace.Wrap(err)
 }
 
+type databaseObjectImportRuleCollection struct {
+	rules []*dbobjectimportrulev1.DatabaseObjectImportRule
+}
+
+func (c *databaseObjectImportRuleCollection) resources() []types.Resource {
+	resources := make([]types.Resource, len(c.rules))
+	for i, b := range c.rules {
+		resources[i] = types.Resource153ToLegacy(b)
+	}
+	return resources
+}
+
+func (c *databaseObjectImportRuleCollection) writeText(w io.Writer, verbose bool) error {
+	t := asciitable.MakeTable([]string{"Name"})
```

Comment:
Could we add some more details on the rule in addition to  "Name"?

Reply:
Done.